test/reference_outputs/misc/parameters_used

```diff
@@ -83,7 +83,7 @@ multichan_opts                          False
 ncores                                  2
 opdir_overwrite                         'overwrite'
 outdir                                  None
-    Value used                          '/home/runner/work/PyBDSF/PyBDSF/test'
+    Value used                          '/home/marcel/code/PyBDSF/test'
```

Comment:
This seems specific to your system.

Reply:
Yep, I can revert it, but it's part of a test that's normally not run in CI.